src/cms/views/regions/region_list_view.py

```diff
@@ -42,11 +43,40 @@ def get(self, request, *args, **kwargs):
         :return: The rendered template response
         :rtype: ~django.template.response.TemplateResponse
         """
+
         regions = Region.objects.all()
+        query = None
+
+        search_data = kwargs.get("search_data")
+        if search_data:
+            search_form = ObjectSearchForm(search_data)
+            if search_form.is_valid():
```

Comment:
Does it make sense to check for `search_form.cleaned_data["query"]` in the `is_valid()` method of `ObjectSearchForm` Then we could also skip the following check here and other places.